numpy/lib/tests/test_histograms.py

```diff
@@ -346,6 +346,15 @@ def test_precision(self):
         self.do_precision(np.single, np.longdouble)
         self.do_precision(np.double, np.longdouble)
 
+    def test_histogram_bin_edges(self):
+        hist, e     = histogram([1, 2, 3, 4], [1, 2])
+        edges       = histogram_bin_edges([1, 2, 3, 4], [1, 2])
+        assert_array_equal(edges, e)
+
+        arr         = np.array([0.,  0.,  0.,  1.,  2.,  3.,  3.,  4.,  5.])
+        hist, e     = histogram(arr, bins=30, range=(-0.5, 5))
+        edges       = histogram_bin_edges(arr, bins=30, range=(-0.5, 5))
```

Comment:
Aligning at equals like this isn't recommended by PEP8, so we tend not to do it in numpy. Otherwise these tests look reasonable. Maybe add one for a string `bins` argument too?

Reply:
Okay, will keep this in mind henceforth. Added another test.